CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### New features
 
+* [#8921](https://github.com/rubocop-hq/rubocop/pull/8921): Prevent `Lint/LiteralInInterpolation` from removing necessary interpolation in `%W[]` and `%I[]` literals. ([@knu][])
```

Comment:
That should go under "Bugs fixed".

Reply:
I fIxed it, and rebased this PR on the latest master.